index.html

```diff
@@ -0,0 +1,461 @@
+                                <img class="follow-user-photo" src="https://th.bing.com/th/id/OIP.fRYqb7px5jByciuIisZyjwHaH0?pid=ImgDet&rs=1" alt="user photo">
+                                <div class="follow-user-details">
+                                    <div class="follow-user-name">Sir Quackers</div>
+                                    <div class="follow-user-handle">@sir_quackers20</div>
+                                </div>
+                                <div class="follow-button">
+                                    <button class="follow-link">Follow</button>
+                                </div>
+                            </a>
+                            <a class="follow-user" href="#">
+                                <img class="follow-user-photo" src="https://yt3.ggpht.com/a/AGF-l78-OMGsDVO-YjQWPkm-w84cKf2SO-qBldcrRA=s900-c-k-c0xffffffff-no-rj-mo" alt="user photo">
+                                <div class="follow-user-details">
+                                    <div class="follow-user-name">Daffy Duck</div>
+                                    <div class="follow-user-handle">@real_daffy_duck</div>
+                                </div>
+                                <div class="follow-button">
+                                    <button class="follow-link">Follow</button>
+                                </div>
+                            </a>
+                            <a class="follow-user" href="#">
+                                <img class="follow-user-photo" src="https://th.bing.com/th/id/OIP.5mniscNIFuD91xUvgCkW5wHaHa?pid=ImgDet&rs=1" alt="user photo">
+                                <div class="follow-user-details">
+                                    <div class="follow-user-name">Donald Duck</div>
+                                    <div class="follow-user-handle">@i_am_donald_duck</div>
+                                </div>
+                                <div class="follow-button">
+                                    <button class="follow-link">Follow</button>
+                                </div>
+                            </a>
+                        </div>
+                        <div class="show-more">
+                            <a class="show-more-link" href="#">Show more</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="messages">
+                    <a class="messages-link" href="">
+                        <div class="messages-grid">
+                            <div class="messages-text">Messages</div>
+                            <img class="new-message" src="./svgs/right-menu-svgs/new-message-icon.svg" alt="new message icon">
+                            <img class="expand" src="./svgs/right-menu-svgs/expand-menu.svg" alt="">
+                        </div> 
+                    </a>
+                </div>
+            </div>
+        </main>
+    </div>
+</body>
+</html>
```

Comment:
The amount of detail youve gone into in your HTML is awesome, and really clear to read too